scenario_specifications.tex

```diff
@@ -1,25 +1,19 @@
 \section{Scenario Specifications}
 
-The simulation is run from year 1970 to 2160. Year 2030 is chosen for the 
-SFR development consideration.
 
 \begin{itemize}
-	\item \textbf{Simulation Time:}  1970-2160 
-	\item \textbf{\gls{SFR} Available Year:}  2030  
+	\item \textbf{Simulation Time:}  1970-2060 
```

Comment:
Again, I don't think you can learn much from a simulation this short. Where is this altered decision coming from?